app/models/admin_menu.rb

```diff
@@ -77,8 +87,14 @@ def self.nested_menu_items(scope_name, nav_item)
 
       items[:children].each do |child|
         return items if child[:controller] == nav_item
+        # NOTE: [@jeremyf] trying to puzzle this one out.  My read is that if the "grandchild" of the
+        # scope matches the controller, return the parent node (e.g. the item).
       end
     end
+
+    # Because we're using each loops, with short-circuiting returns, we need to make sure we don't
+    # retrun the results of `items[:children].each`, which will be `items[:children]`.
```

Comment:
```suggestion
    # Because we're using each loops, with short-circuiting returns, we need to make sure we don't
    # return the results of `items[:children].each`, which will be `items[:children]`.
```